elements/pfe-cta/src/pfe-cta--lightdom.scss

```diff
@@ -95,6 +91,9 @@ $LOCAL: cta;
 }
 
 @media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) { //ie11
+  pfe-cta[pfe-priority="secondary"]  { //ie11
```

Comment:
@castastrophe heads up I moved this style down here, and also modified lines 65 & 66. There was dark text on dark backgrounds in IE11, this resolves that issue

Reply:
@LyndseyR can you move these updates to the lightdom CSS into your pfe-cta branch? then we can keep the scope here to just fixing functions

Reply:
I pulled these changes out into https://github.com/patternfly/patternfly-elements/pull/569